src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/i18n/PluralForms.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.govukfrontend.views.viewmodels.i18n
+
+object PluralForms {
+  type PluralForms = Map[String, String]
```

Comment:
@kyle-bowden I know I was the one who mentioned this, but actually seeing it I'm realising it's providing no value - think we can just remove this and use directly `Map[String, String]` just my brain trying to overcomplicate stuff

Reply:
I do like the idea of giving types like this, as it makes it easier to understand what's happening but I think in this instance its probably just easier to keep it as a Map[String, String]